--- opencv-3.4.1/modules/core/include/opencv2/core/cvdef.h~	2018-02-23 09:38:33.000000000 +0100
+++ opencv-3.4.1/modules/core/include/opencv2/core/cvdef.h	2018-04-27 22:17:38.423275013 +0200
@@ -480,7 +480,7 @@
 using std::int64_t;
 using std::uint64_t;
 }
-#else
+#elif __cplusplus
 #include <stdint.h>
 namespace cv {
 typedef ::int8_t int8_t;
@@ -492,6 +492,16 @@
 typedef ::int64_t int64_t;
 typedef ::uint64_t uint64_t;
 }
+#else
+#include <stdint.h>
+typedef int8_t int8_t;
+typedef uint8_t uint8_t;
+typedef int16_t int16_t;
+typedef uint16_t uint16_t;
+typedef int32_t int32_t;
+typedef uint32_t uint32_t;
+typedef int64_t int64_t;
+typedef uint64_t uint64_t;
 #endif
 #endif
 
--- opencv-3.4.1/modules/highgui/include/opencv2/highgui/highgui_c.h~	2018-02-23 09:38:33.000000000 +0100
+++ opencv-3.4.1/modules/highgui/include/opencv2/highgui/highgui_c.h	2018-04-30 00:00:01.659113349 +0200
@@ -136,7 +136,11 @@
 CVAPI(double) cvGetWindowProperty(const char* name, int prop_id);
 
 /* Get window image rectangle coordinates, width and height */
+#ifdef __cplusplus
 CVAPI(cv::Rect)cvGetWindowImageRect(const char* name);
+#else
+CVAPI(CvRect)cvGetWindowImageRect(const char* name);
+#endif
 
 /* display image within window (highgui windows remember their content) */
 CVAPI(void) cvShowImage( const char* name, const CvArr* image );
