--- opencv-4.5.1/modules/core/include/opencv2/core/cvdef.h~	2020-12-21 21:01:38.000000000 +0100
+++ opencv-4.5.1/modules/core/include/opencv2/core/cvdef.h	2021-03-07 19:39:51.440476065 +0100
@@ -719,6 +719,7 @@
 /****************************************************************************************\
 *                                    C++ 11                                              *
 \****************************************************************************************/
+#ifdef __cplusplus
 #ifndef CV_CXX11
 #  if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1800)
 #    define CV_CXX11 1
@@ -760,6 +761,7 @@
 #ifndef CV_CONSTEXPR
 #  define CV_CONSTEXPR
 #endif
+#endif
 
 // Integer types portatibility
 #ifdef OPENCV_STDINT_HEADER
