--- opencv-3.1.0/opencv_contrib-3.1.0/modules/tracking/include/opencv2/tracking/onlineMIL.hpp~	2015-12-17 18:11:31.000000000 +0100
+++ opencv-3.1.0/opencv_contrib-3.1.0/modules/tracking/include/opencv2/tracking/onlineMIL.hpp	2017-12-13 08:59:51.077710225 +0100
@@ -54,8 +54,6 @@
 //TODO based on the original implementation
 //http://vision.ucsd.edu/~bbabenko/project_miltrack.shtml
 
-#define  sign(s)  ((s > 0 ) ? 1 : ((s<0) ? -1 : 0))
-
 class ClfOnlineStump;
 
 class CV_EXPORTS ClfMilBoost
--- opencv-3.1.0/opencv_contrib-3.1.0/modules/tracking/src/onlineMIL.cpp~	2015-12-17 18:11:31.000000000 +0100
+++ opencv-3.1.0/opencv_contrib-3.1.0/modules/tracking/src/onlineMIL.cpp	2017-12-13 09:00:15.158067835 +0100
@@ -42,6 +42,8 @@
 #include "precomp.hpp"
 #include "opencv2/tracking/onlineMIL.hpp"
 
+#define  sign(s)  ((s > 0 ) ? 1 : ((s<0) ? -1 : 0))
+
 template<class T> class SortableElementRev
 {
  public:
