--- OpenCV-2.3.1/OpenCVFindXimea.cmake.orig	2011-09-12 20:45:38.000000000 +0200
+++ OpenCV-2.3.1/OpenCVFindXimea.cmake	2012-02-01 19:18:07.619900166 +0100
@@ -14,15 +14,13 @@
 set(XIMEA_PATH)
 set(XIMEA_LIBRARY_DIR)
 
-# Try to find the XIMEA API path in registry.
-GET_FILENAME_COMPONENT(XIMEA_PATH "[HKEY_CURRENT_USER\\Software\\XIMEA\\CamSupport\\API;Path]" ABSOLUTE)                                                
+find_file(XIMEA_INC "m3Api.h" PATHS $ENV{XIMEA_INCLUDE} "/usr/include/ximea")
+find_library(XIMEA_LIB "m3api" PATHS $ENV{XIMEA_LIB} "/usr/lib${LIB_SUFFIX}")
 
-if(XIMEA_PATH)
+if(XIMEA_INC AND XIMEA_LIB)
+  get_filename_component(XIMEA_LIBRARY_DIR "${XIMEA_LIB}" PATH CACHE)
+  get_filename_component(XIMEA_PATH "${XIMEA_INC}" PATH CACHE)
   set(XIMEA_FOUND 1)
-  
-  # set LIB folders
-  set(XIMEA_LIBRARY_DIR "${XIMEA_PATH}\\x86")
-  
 else()
   set(XIMEA_FOUND 0)
 endif()                                            
--- OpenCV-2.3.1/modules/highgui/src/cap_ximea.cpp.orig	2011-09-12 20:41:29.000000000 +0200
+++ OpenCV-2.3.1/modules/highgui/src/cap_ximea.cpp	2012-02-01 20:12:54.750010230 +0100
@@ -1,4 +1,4 @@
-
+#include <cstdio>
 #include "precomp.hpp"
 
 #include "xiApi.h"
@@ -245,9 +245,7 @@
 
 void CvCaptureCAM_XIMEA::errMsg(char* msg, int errNum)
 {
-	char buf[512];
-	sprintf( buf, "%s : %d\n", msg, errNum);
-	OutputDebugString(buf);
+	fprintf(stderr, "%s : %d\n", msg, errNum);
 }
 
-/**********************************************************************************/
\ No newline
+/**********************************************************************************/
