--- opencv-2.4.8/cmake/OpenCVFindLibsVideo.cmake.orig	2014-02-11 15:18:37.396156598 +0100
+++ opencv-2.4.8/cmake/OpenCVFindLibsVideo.cmake	2014-02-11 15:19:54.716153353 +0100
@@ -127,6 +127,7 @@
 if(WITH_V4L)
   if(WITH_LIBV4L)
     CHECK_MODULE(libv4l1 HAVE_LIBV4L)
+    CHECK_MODULE(libv4l2 HAVE_LIBV4L2)
   endif()
   CHECK_INCLUDE_FILE(linux/videodev.h HAVE_CAMV4L)
   CHECK_INCLUDE_FILE(linux/videodev2.h HAVE_CAMV4L2)
