--- OpenCV-2.4.3/cmake/OpenCVFindXimea.cmake.orig	2012-10-18 00:55:38.000000000 +0200
+++ OpenCV-2.4.3/cmake/OpenCVFindXimea.cmake	2013-02-14 19:10:24.278758230 +0100
@@ -27,10 +27,10 @@
     set(XIMEA_FOUND 0)
   endif()
 else()
-  if(EXISTS /opt/XIMEA)
+  if(EXISTS /usr/include/ximea)
     set(XIMEA_FOUND 1)
     # set folders
-    set(XIMEA_PATH /opt/XIMEA/include)
+    set(XIMEA_PATH /usr/include/ximea)
   else()
     set(XIMEA_FOUND 0)
   endif()
