--- opencv-4.5.1/modules/videoio/cmake/detect_ximea.cmake~	2020-12-21 21:01:38.000000000 +0100
+++ opencv-4.5.1/modules/videoio/cmake/detect_ximea.cmake	2021-03-06 09:50:48.915720952 +0100
@@ -13,6 +13,7 @@
     set(lib_suffix "32")
   endif()
   find_path(XIMEA_INCLUDE "xiApi.h"
+    "/usr/include/ximea"
     PATHS "${XIMEA_ROOT}" ENV XIMEA_ROOT "/opt/XIMEA"
     HINTS "${regpath}"
     PATH_SUFFIXES "include" "API")
