--- opencv-4.5.5/modules/gapi/cmake/DownloadADE.cmake.orig	2021-12-25 04:53:27.000000000 +0100
+++ opencv-4.5.5/modules/gapi/cmake/DownloadADE.cmake	2023-07-08 18:29:26.487465343 +0200
@@ -1,7 +1,7 @@
 set(ade_src_dir "${OpenCV_BINARY_DIR}/3rdparty/ade")
-set(ade_filename "v0.1.1f.zip")
-set(ade_subdir "ade-0.1.1f")
-set(ade_md5 "b624b995ec9c439cbc2e9e6ee940d3a2")
+set(ade_filename "v0.1.2a.zip")
+set(ade_subdir "ade-0.1.2a")
+set(ade_md5 "fa4b3e25167319cb0fa9432ef8281945")
 ocv_download(FILENAME ${ade_filename}
              HASH ${ade_md5}
              URL
