--- configure.in.org	2009-03-18 23:03:33.397335774 +0100
+++ configure.in	2009-03-18 23:04:10.016786290 +0100
@@ -612,7 +612,7 @@
        with_ffmpeg=yes
     fi
     if test x"$with_ffmpeg" = "xyes"; then
-        AC_CHECK_HEADER(ffmpeg/avcodec.h,
+        AC_CHECK_HEADER(libavcodec/avcodec.h,
                         AC_SEARCH_LIBS([avcodec_decode_video],[avcodec_pic avcodec],                                      [
                                        AC_SEARCH_LIBS([av_open_input_file], [avformat_pic avformat],
                                                     [
@@ -625,7 +625,7 @@
                                      ],
                                      [ have_ffmpeg=no ]),
                         [ have_ffmpeg=no ])
-        AC_CHECK_HEADER([ffmpeg/swscale.h],
+        AC_CHECK_HEADER([libswscale/swscale.h],
                             AC_CHECK_LIB(swscale,sws_getContext,
                                          [
 					   AC_DEFINE(HAVE_FFMPEG_SWSCALE,,[ffmpeg's libswscale])
--- otherlibs/highgui/cvcap_ffmpeg.cpp~	2008-09-29 22:49:15.000000000 +0200
+++ otherlibs/highgui/cvcap_ffmpeg.cpp	2009-03-18 23:08:26.526923830 +0100
@@ -53,10 +53,10 @@
 #include <errno.h>
 #endif
 
-#include <ffmpeg/avformat.h>
-#include <ffmpeg/avcodec.h>
+#include <libavformat/avformat.h>
+#include <libavcodec/avcodec.h>
 #if defined(HAVE_FFMPEG_SWSCALE)
-#include <ffmpeg/swscale.h>
+#include <libswscale/swscale.h>
 #endif
 }
 
