diff -ur opencv-1.1.0.orig/samples/c/Makefile.am opencv-1.1.0/samples/c/Makefile.am
--- opencv-1.1.0.orig/samples/c/Makefile.am	2009-07-02 16:29:14.088536975 +0200
+++ opencv-1.1.0/samples/c/Makefile.am	2009-07-02 16:30:05.145362986 +0200
@@ -7,8 +7,8 @@
 right01.jpg right02.jpg right03.jpg right04.jpg right05.jpg right06.jpg right07.jpg \
 right08.jpg right09.jpg right11.jpg right12.jpg right13.jpg right14.jpg \
 stereo_calib.txt build_all.sh cvsample.dsp makefile.ms makefile.gnu \
-agaricus-lepiota.data letter-recognition.data bgfg_codebook.cpp bgfg_segm.cpp blobtrack.cpp camshiftdemo.c \
-contours.c convexhull.c camshiftdemo.c contours.c convert_cascade.c convexhull.c \
+agaricus-lepiota.data letter-recognition.data bgfg_codebook.cpp bgfg_segm.cpp blobtrack.cpp \
+camshiftdemo.c contours.c convert_cascade.c convexhull.c \
 delaunay.c demhist.c dft.c distrans.c drawing.c edge.c facedetect.c \
 facedetect.cmd ffilldemo.c find_obj.cpp fitellipse.c houghlines.c kalman.c kmeans.c image.cpp \
 inpaint.cpp laplace.c letter_recog.cpp lkdemo.c minarea.c morphology.c motempl.c \
