--- OpenCV-2.4.3/modules/ocl/perf/perf_gemm.cpp.orig	2012-10-12 00:55:12.000000000 +0200
+++ OpenCV-2.4.3/modules/ocl/perf/perf_gemm.cpp	2013-02-16 10:09:54.859534100 +0100
@@ -109,5 +109,5 @@
 INSTANTIATE_TEST_CASE_P(ocl_gemm, Gemm, testing::Combine(
                             testing::Values(CV_32FC1, CV_32FC2/* , CV_64FC1, CV_64FC2*/),
                             testing::Values(cv::Size(512, 512), cv::Size(1024, 1024)),
-                            testing::Values(0, cv::GEMM_1_T, cv::GEMM_2_T, cv::GEMM_1_T + cv::GEMM_2_T)));
-#endif
\ No newline
+                            testing::Values(0, int(cv::GEMM_1_T), int(cv::GEMM_2_T), cv::GEMM_1_T + cv::GEMM_2_T)));
+#endif
