--- OpenCV-2.2.0/modules/highgui/CMakeLists.txt~	2010-12-05 04:35:24.000000000 +0100
+++ OpenCV-2.2.0/modules/highgui/CMakeLists.txt	2010-12-07 13:43:58.544879668 +0100
@@ -302,8 +302,8 @@
 
 install(TARGETS ${the_target}
 	RUNTIME DESTINATION bin COMPONENT main
-	LIBRARY DESTINATION lib COMPONENT main
-	ARCHIVE DESTINATION lib COMPONENT main)
+	LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT main
+	ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT main)
 
 install(FILES ${highgui_ext_hdrs}
         DESTINATION include/opencv2/highgui
--- OpenCV-2.2.0/modules/gpu/CMakeLists.txt~	2010-12-07 13:41:30.061546334 +0100
+++ OpenCV-2.2.0/modules/gpu/CMakeLists.txt	2010-12-07 13:44:31.161546335 +0100
@@ -124,8 +124,8 @@
 
 install(TARGETS ${the_target}
 	RUNTIME DESTINATION bin COMPONENT main
-	LIBRARY DESTINATION lib COMPONENT main
-	ARCHIVE DESTINATION lib COMPONENT main)
+	LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT main
+	ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT main)
 
 install(FILES ${lib_hdrs}
 	DESTINATION include/opencv2/${name}
--- OpenCV-2.2.0/modules/ffmpeg/CMakeLists.txt~	2010-12-05 04:35:24.000000000 +0100
+++ OpenCV-2.2.0/modules/ffmpeg/CMakeLists.txt	2010-12-07 13:45:03.004879667 +0100
@@ -49,6 +49,6 @@
 
 install(TARGETS ${the_target}
     RUNTIME DESTINATION bin COMPONENT main
-    ARCHIVE DESTINATION lib COMPONENT main
-    LIBRARY DESTINATION lib COMPONENT main
+    ARCHIVE DESTINATION lib${LIB_SUFFIX} COMPONENT main
+    LIBRARY DESTINATION lib${LIB_SUFFIX} COMPONENT main
     )
--- OpenCV-2.2.0/CMakeLists.txt~	2010-12-07 13:49:56.981546334 +0100
+++ OpenCV-2.2.0/CMakeLists.txt	2010-12-07 13:50:17.114879668 +0100
@@ -513,8 +513,8 @@
 
     string(REGEX MATCH "[0-9].[0-9]" PYTHON_VERSION_MAJOR_MINOR "${PYTHON_VERSION_FULL}")
     if(UNIX)
-        set(PYTHON_PLUGIN_INSTALL_PATH lib/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages/opencv)
-        set(PYTHON_PACKAGES_PATH lib/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages)
+	set(PYTHON_PLUGIN_INSTALL_PATH lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages/opencv)
+	set(PYTHON_PACKAGES_PATH lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages)
     endif()
     if(WIN32)
         get_filename_component(PYTHON_PATH "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore\\${PYTHON_VERSION_MAJOR_MINOR}\\InstallPath]" ABSOLUTE CACHE)
