--- OpenCV-2.3.1/CMakeLists.txt~	2011-09-20 16:37:58.864889717 +0200
+++ OpenCV-2.3.1/CMakeLists.txt	2011-09-21 09:33:29.341296893 +0200
@@ -666,9 +666,9 @@
                     OUTPUT_VARIABLE PYTHON_STD_PACKAGES_PATH
                     OUTPUT_STRIP_TRAILING_WHITESPACE)
     if("${PYTHON_STD_PACKAGES_PATH}" MATCHES "site-packages")
-        set(PYTHON_PACKAGES_PATH lib/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages CACHE PATH "Where to install the python packages.")
+	set(PYTHON_PACKAGES_PATH lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR_MINOR}/site-packages CACHE PATH "Where to install the python packages.")
     else() #debian based assumed, install to the dist-packages.
-        set(PYTHON_PACKAGES_PATH lib/python${PYTHON_VERSION_MAJOR_MINOR}/dist-packages CACHE PATH "Where to install the python packages.")
+	set(PYTHON_PACKAGES_PATH lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR_MINOR}/dist-packages CACHE PATH "Where to install the python packages.")
     endif()
 endif()
 if(CMAKE_HOST_WIN32)
