--- opencv-2.4.11/cmake/OpenCVCompilerOptions.cmake~	2015-05-19 11:39:16.000000000 +0300
+++ opencv-2.4.11/cmake/OpenCVCompilerOptions.cmake	2015-05-19 11:47:57.851517663 +0300
@@ -47,6 +47,8 @@
   endif()
 endmacro()
 
+add_extra_compiler_option(-D__STDC_CONSTANT_MACROS)
+
 if(MINGW)
   # http://gcc.gnu.org/bugzilla/show_bug.cgi?id=40838
   # here we are trying to workaround the problem
@@ -110,10 +110,6 @@
     add_extra_compiler_option(-Werror)
   endif()
 
-  if(X86 AND NOT MINGW64 AND NOT X86_64 AND NOT APPLE)
-    add_extra_compiler_option(-march=i686)
-  endif()
-
   if(APPLE)
     add_extra_compiler_option(-Wno-semicolon-before-method-body)
   endif()
